services/FileDelete.proto

```diff
@@ -27,7 +27,7 @@ option java_multiple_files = true;
 
 import "BasicTypes.proto";
 
-/* Delete the given file. After deletion, it will be marked as deleted and will have no contents. But information about it will continue to exist until it expires. A list of keys  was given when the file was created. All the keys on that list must sign transactions to create or modify the file, but any single one of them can be used to delete the file. Each "key" on that list may itself be a threshold key containing other keys (including other threshold keys). */
+/* Delete the given file. After deletion, it will be marked as deleted and will have no contents. But information about it will continue to exist until it expires. A list of keys was given when the file was created. All the top level keys on that list must sign transactions to create or modify the file, but any single one of the top level keys can be used to delete the file. In this instance, the top level keys in a key list are operating as a 1-of-M signing requirement. Each "key" on that list may itself be a threshold key containing other keys (including other threshold keys). */
```

Comment:
Suggest to modify "A list of keys was given when the file was created. All the top level keys on that list must sign transactions to create or modify the file, but any single one of the top level keys can be used to delete the file. In this instance, the top level keys in a key list are operating as a 1-of-M signing requirement. Each "key" on that list may itself be a threshold key containing other keys (including other threshold keys)." to "This transaction must be signed by 1-of-M KeyList keys unless keys contains additional KeyList or ThresholdKey and then 1-of-M secondary KeyList or ThresholdKey signing reequipments must be meet also."